cli/wallet/wallet.go

```diff
@@ -517,6 +524,13 @@ func transferAsset(ctx *cli.Context) error {
 		return cli.NewExitError(err, 1)
 	}
 
+	if remark14 != "" {
+		tx.Attributes = append(tx.Attributes, transaction.Attribute{
```

Comment:
In C# example they also add verification (`Script`) and nonce (`Remark1`) attributes to the transaction, although it's not required by the guide. Should we do the same?

Reply:
`Script` is for additional signatures IIRC (like when you're transferring NEP-5 via Invocation transaction), it's not needed for simple UTXO transfers. Nonce is also more relevant for NEP-5 transfers (where you're likely to have the same transaction hash otherwise for a number of transactions), UTXO doesn't need it.